sdk/servicebus/Azure.Messaging.ServiceBus/stress/src/Roles/SessionSender.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus;
+
+namespace Azure.Messaging.ServiceBus.Stress;
+
+/// <summary>
+///   The role responsible for running a <see cref="ServiceBusSender" \> with sessions, and testing its performance over
```

Comment:
```suggestion
///   The role responsible for running a <see cref="ServiceBusSender" /> with sessions, and testing its performance over
```

These tag closing slashes are backwards.   I'm surprised the compiler didn't flag them as invalid XML.   If the compiler is happy, I'm not particularly concerned, but...